modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -453,7 +510,7 @@ const EditFDSFieldModalContent = ({
 						<ClayDropDown.Item
 							className="align-items-center d-flex justify-content-between"
 							key={cellRenderer.value}
-							onClick={() => onItemClick(cellRenderer.label)}
+							onClick={() => onItemClick(cellRenderer.value)}
```

Comment:
Nice! This was a bug cc @kresimir-coko

Reply:
That was a carefully placed booby trap 😄 

<img src="https://media4.giphy.com/media/Z1LYiyIPhnG9O/giphy.gif"/>